diff -ruN templates/author/submission/status.tpl plugins/generic/pid/2.3.x/templates/author/submission/status.tpl
--- templates/author/submission/status.tpl	2011-02-15 15:19:24.000000000 -0700
+++ plugins/generic/pid/2.3.x/templates/author/submission/status.tpl	2011-02-15 15:33:33.000000000 -0700
@@ -12,6 +12,7 @@
 <h3>{translate key="common.status"}</h3>
 
 <table width="100%" class="data">
+	{call_hook name="Template::Author::Submission::Status"}
 	<tr>
 		{assign var="status" value=$submission->getSubmissionStatus()}
 		<td width="20%" class="label">{translate key="common.status"}</td>
diff -ruN templates/sectionEditor/submission/status.tpl plugins/generic/pid/2.3.x/templates/sectionEditor/submission/status.tpl
--- templates/sectionEditor/submission/status.tpl	2011-02-15 15:19:56.000000000 -0700
+++ plugins/generic/pid/2.3.x/templates/sectionEditor/submission/status.tpl	2011-02-15 15:33:50.000000000 -0700
@@ -10,6 +10,7 @@
 <h3>{translate key="common.status"}</h3>
 
 <table width="100%" class="data">
+	{call_hook name="Template::sectionEditor::Submission::Status"}
 	<tr>
 		{assign var="status" value=$submission->getSubmissionStatus()}
 		<td width="20%" class="label">{translate key="common.status"}</td>
diff -ruN lib/pkp/classes/oai/OAI.inc.php plugins/generic/pid/2.3.x/lib/pkp/classes/oai/OAI.inc.php
--- lib/pkp/classes/oai/OAI.inc.php	2011-02-15 14:54:37.000000000 -0700
+++ plugins/generic/pid/2.3.x/lib/pkp/classes/oai/OAI.inc.php	2011-02-15 15:35:21.000000000 -0700
@@ -256,6 +256,7 @@
 			"\t\t</record>\n" .
 			"\t</GetRecord>\n";
 
+		HookRegistry::call('OAIDAOinc::_getRecord', array(&$response, &$metadataPrefix, &$record, &$this->params));
 		$this->response($response);
 	}
 
@@ -543,6 +544,7 @@
 
 		$response .= "\t</ListRecords>\n";
 
+		HookRegistry::call('OAIDAOinc::_listRecords', array(&$response, &$metadataPrefix, &$records, &$this->params));
 		$this->response($response);
 	}
 
diff -ruN templates/article/article.tpl plugins/generic/pid/2.3.x/article.tpl
--- templates/article/article.tpl	2012-01-17 12:46:46.000000000 -0700
+++ plugins/generic/pid/2.3.x/article.tpl	2012-01-17 12:45:29.000000000 -0700
@@ -126,7 +126,7 @@
 		{/if}
 	{/if}
 {/if}
-
+{call_hook name="Template::Article::PID"}
 {include file="article/comments.tpl"}
 
 {include file="article/footer.tpl"}